runtime/cudaq/platform/default/rest_server/helpers/server_impl/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+include(FetchContent)
+FetchContent_Declare(
+    asio
+    GIT_REPOSITORY    https://github.com/chriskohlhoff/asio.git
+    GIT_TAG           asio-1-28-2
+)
+FetchContent_GetProperties(asio)
+if(NOT asio_POPULATED)
+  FetchContent_Populate(asio)
+endif()
+
+FetchContent_Declare(
+    crow_cpp
+    GIT_REPOSITORY    https://github.com/CrowCpp/Crow.git
```

Comment:
Should this be added to our `tpls` directory as a submodule?

Reply:
Same as https://github.com/NVIDIA/cuda-quantum/pull/1012#discussion_r1449441051

Reply:
OK, that's fair enough. We just need to make sure it is added to our dependency list. @bettinaheim